lib/kubernetes/constants.go

```diff
@@ -16,5 +16,13 @@ limitations under the License.
 
 package kubernetes
 
-// AllNamespaces can be used to query pods in all namespaces.
-const AllNamespaces = ""
+const (
+	// AllNamespaces can be used to query pods in all namespaces.
+	AllNamespaces = ""
+
+	// NamespaceKubeSystem specifies kube-system namespace.
+	NamespaceKubeSystem = "kube-system"
+
+	// NamespaceMonitoring specifies monitoring namespace.
+	NamespaceMonitoring = "monitoring"
```

Comment:
I don't think we should hardcode namespaces for satellite to check. Let planet-agent pass them to satellite via checker configuration.

Reply:
Updated in 7049874